types/foundry-vtt-api/types/apps/sidebar/sidebartab.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * An abstract pattern followed by the different tabs of the sidebar
+ */
+declare class SidebarTab extends Application {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/sidebar/sidebartab.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.